docs/running-headscale-linux.md

```diff
@@ -78,6 +84,13 @@ Verify `headscale` is available:
 curl http://127.0.0.1:9090/metrics
 ```
 
+Verify you can issue commands to `headscale`:
+
+```shell
+headscale users list
+```
+When running headscale as a normal user it is possible that the metrics page is available but you cannot issue commands to headscale. In that case you need to change to the directory from where you started headscale.
```

Comment:
Here it is not obvious what the last sentence intends to convey.

Reply:
When I was trying to test my configuration for the first time I ran headscale from `/etc/headscale` as a non-privileged user. Then according to the docs I opened a second terminal (first was busy running headscale) and I wanted to issue a command, `headscale users list`. It did not work. It only works if I `cd /etc/headscale` in the second terminal.
I tested some more and `/etc/headscale` is not the special case, as a normal user any directory where you start headscale is the only one from where you can issue commands to it.

I don't think that this behaviour is intended, and also mostly people run it from systemd or as a root user, neither case has any problems. But I think it is not unreasonable to test your config as a normal user so I think this quirk should be documented. I would have been happy if I didn't have to figure it out myself.

Maybe a senctence like: In that case you need to change to the directory in the second terminal where (you issue the command) to the same directory from where you started headscale in the first terminal.

I'm not a great writer as you can see, if you have a clearer explanation in mind I'm very open to it!

Reply:
I think the reason behind why you needed to be in the directory is slightly incorrect.

I think that you need to make sure that `/etc/headscale/config.yaml` is reachable for the unprivileged user, it will need to know how to find the headscale socket, which is located in this config.

The error might have been that you could read the config, by being in the same dir, but you could not access the actual directory with the config (`/etc/headscale`).

Reply:
I just talked to a user on discord. In his case he was running the headscale server as root and was running the headscale users command as root and it did not work, only when he cd-ed to `/etc/headscale`. I don't think permissions could be an issue in this case.... Maybe this should be a separate issue.